app/models/host.rb

```diff
@@ -90,7 +90,10 @@ class Host < ApplicationRecord
                             :inverse_of => :host
   has_many                  :host_aggregate_hosts, :dependent => :destroy
   has_many                  :host_aggregates, :through => :host_aggregate_hosts
-
+  
+  # Physical infra reference
+  has_one :physical_server, :foreign_key => "serial_number", :primary_key => "service_tag", :class_name => "PhysicalServer"
```

Comment:
This relationship is correct, in that a `Host has_one :physical_server`.  However, the foreign key of `"serial_number"` is incorrect.

This should just be:
```ruby
has_one :physical_server, :inverse_of => :host
```